protected/humhub/widgets/form/ActiveField.php

```diff
@@ -155,4 +161,20 @@ public function radioList($items, $options = [])
 
         return parent::radioList($items, $options);
     }
+
+    /**
+     * @inheritdoc
+     */
+    protected function addAriaAttributes(&$options)
+    {
+        $this->setReadOnlyFromOptions($options);
+        parent::addAriaAttributes($options);
+    }
+
+    protected function setReadOnlyFromOptions(array $options): void
```

Comment:
Can we make this private?

Reply:
@luke- Ok, we can, at this moment I don't find where the class is extended, I just did it protected to avoid restriction for future extended classes.

Reply:
https://github.com/humhub/humhub/pull/7708/commits/10515bb38f9fd589515c5be9c86473e96705884b

Reply:
I think we should be rather restrictive from the beginning. If necessary, we can always relax the rules later.